src/install/archive/web/download_archive.dart

```diff
@@ -95,9 +97,9 @@ Future getListing(String channel, String respString) async {
   // Format is lines of "channels/stable/release/\d+/".
   Iterable<Future> versionRequests = versions.map((String path) async {
     try {
+      // XXX: We get 403/404s for dev/release/1.XY.0 where XY > 16.
```

Comment:
I know why this is.  We were requested to copy the stable versions to the dev channel, so that the dev channel would have 1.x.0-dev.foo followed by 1.x.0 followed by 1.x+1.0-dev.foo.  And when copying the files over, I think we missed giving the READ: ALLUSERS ACL to the new copy.  We could fix this.